source/change-log/index.html.md.erb

```diff
@@ -7,6 +7,27 @@ description: Find out what has changed in the specification and make sure your a
 
 # Change log
 
+### Version 3.2
+
+#### **&lt;Release date TBC&gt;**
+
+ * For the mobile application via server and mobile application direct connection methods, you do not need to provide a
+   value for `os` in Gov-Client-User-IDs. If your application submits values for `os` it is not an issue.
+ * For all connection methods other than web application via server, mobile application via server and mobile
+   application direct, you still need to submit a value for `os` in Gov-Client-User-IDs.
+ * For the mobile application via server and mobile application direct connection methods, you do not need to submit
+   values for Gov-Client-MAC-Addresses. If your application submits values for this header it is not an issue.
+ * For all connection methods other than web application via server, mobile application via server and mobile
+   application direct, you still need to submit values for Gov-Client-MAC-Addresses.
+ * Clarification that Gov-Vendor-Forwarded must contain at least 1 pair of IPs with Gov-Client-Public-IP in the ‘for’ field and
+   Gov-Vendor-Public-IP in the ‘by’ field.
+ * Clarification that for the desktop application via server, mobile application via server and other via server connection methods,
+   the value submitted for Gov-Vendor-Version must contain at least two key value pairs.
+ * Clarification that for the web application via server connection method, you must submit the value `false` if the user’s browser does
+   not support Do Not Track.
+
+This will be updated in the Directions. To check what you need to send, <a href="../connection-method/">select your connection method</a>.
+
```

Comment:
I think this should be more concise. I.e. in version 3.1 we clarified the format for Gov-Client-Timezone header so the change log only mentions this rather than repeating the clarification. 

So the mobile `os` field update could be (in line with the 2022.01.31 update):

```
For the mobile application via server and mobile application direct connection methods, you do not need to provide a value for `os` in Gov-Client-User-IDs. If your application submits values for `os` it is not an issue.
```

Equally, I think we should drop:
```
For all connection methods other than web application via server, mobile application via server and mobile application direct, you still need to submit values for Gov-Client-MAC-Addresses.
```

The point on Gov-Vendor-Forwarded suggests these two public IP addresses should be in the same pair.

Last one could just be
```
Clarification on Gov-Client-Browser-Do-Not-Track
```

Reply:
Content updated following meeting.